README.md

```diff
@@ -6,9 +6,9 @@ The repro can then be used as the basis of a bug report or a starting point for
 
 ## Usage
 
-First, build your project with `--generateTrace traceDir`
+First, build your project with `tsc --generateTrace traceDir`
 
-For a sorted list of compilation hot-spots, run `npx analyze-trace traceDir`
+For a sorted list of compilation hot-spots, run `npx @typescript/analyze-trace traceDir`
```

Comment:
If you install `@typescript/analyze-trace` first, does the original command work for you?  If so, what about adding a step before this one installing the package and then leaving this as-is?

Reply:
The updated command gave this result on my setup 
<img width="340" alt="image" src="https://user-images.githubusercontent.com/434607/153260649-88d88225-d832-4246-a85a-d725b03cd73d.png">
Installing the package globally, then running the original command has the same result
<img width="246" alt="image" src="https://user-images.githubusercontent.com/434607/153260906-6064aa50-894b-463b-bd98-4f5e308da5ef.png">

Initially, I thought it was an error (as in, not reading the file), but checking the code I realize this output is probably due to `interestingCount` being 0, as in the script didn't detect anything worthwhile reporting.

Reply:
@rodelta `interestingCount` is incremented if a hotspot or other problem is found in a project.  All that check does is print out an extra line confirming that the projects were, in fact, reviewed.

It's quite normal to find nothing of interest.  I suspect that if you view the trace directly (i.e. with `about://tracing`) you'll see that there aren't any "wide" stacks.  Typically, this just means that the cost is spread evenly across many operations, in which case tracing won't be very helpful as a profiling technique and you should switch to something like [pprof-it](https://www.npmjs.com/package/pprof-it).

You may also wish to try playing with the various threshold options (pass `--help` to learn more) to force more verbose output but, personally, I rarely find it worthwhile to investigate anything not flagged by the default settings.